core/domain/exp_services_test.py

```diff
@@ -881,13 +881,102 @@ def test_export_by_versions(self):
 def _get_change_list(state_name, property_name, new_value):
     """Generates a change list for a single state change."""
     return [{
-        'cmd': 'edit_state_property',
+        'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
         'state_name': state_name,
         'property_name': property_name,
         'new_value': new_value
     }]
 
 
+class UpdateGadgetTests(ExplorationServicesUnitTests):
+    """Test updating a gadget."""
+
+    def setUp(self):
+        super(UpdateGadgetTests, self).setUp()
+        exploration = self.save_new_valid_exploration(
+            self.EXP_ID, self.owner_id)
+
+        self.init_state_name = exploration.init_state_name
+
+        self.param_changes = [{
```

Comment:
Yes @BenHenning  It is because it initializes the exploration every time before a test in the class is run, without this, the tests will not be ran (an error will occur)